src/main/java/org/apache/commons/dbutils/BeanProcessor.java

```diff
@@ -465,9 +465,6 @@ protected Method getWriteMethod(final Object target, final PropertyDescriptor pr
 
         for (int col = 1; col <= cols; col++) {
             String columnName = rsmd.getColumnLabel(col);
-            if (null == columnName || 0 == columnName.length()) {
```

Comment:
How about using "columnName.isEmpty()" instead of "0 == columnName.length()"
May you please add a unit test?

Reply:
This whole patch seems redundant with what JDBC specifies as the behavior for getColimnLabel(). What am I missing? This patch is workaround for buggy JDBC drivers?